dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleClientHandler.java

```diff
@@ -90,6 +95,13 @@ private void writeRequest(ChannelHandlerContext ctx, final Request req, final Ch
         if (StringUtils.isNotEmpty(ssl)) {
             ctx.channel().attr(TripleConstant.SSL_ATTRIBUTE_KEY).set(Boolean.parseBoolean(ssl));
         }
+
+        // get compressor
+        String compressorStr = ConfigurationUtils.getGlobalConfiguration(ApplicationModel.defaultModel()).getString(COMPRESSOR_KEY, DEFAULT_COMPRESSOR);
```

Comment:
Obtain the configuration in multi-instance mode from url

Reply:
done